modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -697,7 +697,7 @@ export default class LargeVideoManager {
             APP.store.dispatch(updateKnownLargeVideoResolution(height));
         }
 
-        const currentAspectRatio = width / height;
+        const currentAspectRatio = height === 0 ? 0 : width / height;
```

Comment:
does it make sense to resize if either height or width change to zero?

Reply:
Practically resize will not do anything. But IMHO it is up to the resize to decide what to do with the values. That's why I didn't add additional checks here.